### ISSUE TITLE
Create reproducible release versions and archive them

### ISSUE THREAD

**Issue by frauzufall** [fiji/fiji#223]
As discussed [in the forum](https://forum.image.sc/t/fiji-imagej-release-versions/24700), it would be great to have more regular Life-Line Fiji versions / reproducible releases and also archive Fiji each time a new version is uploaded to the [download page](https://imagej.net/Downloads).

Open tasks copied from the forum:

- [ ] Core ImageJ and Fiji update sites fully driven by releases of net.imagej:imagej and sc.fiji:fiji, respectively. No one allowed to upload to the core sites directly. Otherwise, people “patch in” things on top, and the version number reported in the ImageJ status bar no longer reflects the reality of the installation.

- [ ] Consider whether/how to report the version of sc.fiji:fiji in Fiji installations. Otherwise, there is no easy number summarizing the state of the Fiji components.

- [ ] Fix, or work around, the showstopper Maven bug that causes depending on net.imagej:imagej to inherit different versions than the ones it itself believes it depends on due to the pom-scijava dependency management. See:
        https://stackoverflow.com/q/45041888/1207769
        https://issues.apache.org/jira/browse/MNG-5761
        https://gist.github.com/ctrueden/d058330c8a3687317806ce8cc18332c3

Related issues:
https://github.com/imagej/imagej/issues/135
https://github.com/imagej/imagej/issues/178
https://github.com/imagej/imagej/issues/69

**Comment by hubutui** [comment]
Any updates? I check https://downloads.imagej.net/fiji/archive/, and there are no files at all. It would be great it there is a release version for fiji. Currently, the latest release in https://github.com/fiji/fiji/releases is 2.0.0-pre-10 on 28 Dec 2018, but the latest binary from https://downloads.imagej.net/fiji/latest/ is updated on 11 Nov 2019. Are they the same version?

**Comment by ctrueden** [comment]
@hubutui Thanks for pointing out that https://downloads.imagej.net/fiji/archive/ was empty.

TL;DR: Fixed now!

Given that fiji/fiji-builds#2 was merged, that folder should have started to accrue archived versions. I did some digging. The [Travis CI job](https://travis-ci.com/fiji/fiji-builds) _was_ squirreling away the versions into an `archive/` folder... just not _the_ `archive/` folder being served by https://downloads.imagej.net/fiji/archive. I have now fixed this on the server side.

Note that these archived versions are not _releases_ in the sense of stable artifacts deployed to Maven—ultimately that's where we want to go, but for now the `2.0.0-pre-XXX` Maven artifacts need to be cut manually. I'm planning to do `2.0.0-pre-11` very soon.

**Comment by hubutui** [comment]
@ctrueden Hey, I check https://downloads.imagej.net/fiji/archive/20191220-2112/, and there is only zip files for Linux, no tarballs. Anything wrong?

**Comment by ctrueden** [comment]
@hubutui Maybe because the [build that day](https://travis-ci.com/fiji/fiji-builds/builds/141974655) failed. And then nothing has changed since then?
